rhino/src/main/java/org/mozilla/javascript/AccessorSlot.java

```diff
@@ -113,10 +113,34 @@ Function getGetterFunction(String name, Scriptable scope) {
         return getter.asGetterFunction(name, scope);
     }
 
+    @Override
+    boolean isSameGetterFunction(Object function) {
+        if (function == Scriptable.NOT_FOUND) {
+            return true;
+        } else if (getter == null) {
```

Comment:
Maybe a matter of taste - but there is no else needed because of the return

Reply:
Changed.